tests/Unit/ControlSystem/Test_FuntionOfTimeUpdater.cpp

```diff
@@ -86,3 +86,77 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.FunctionOfTimeUpdater.Translation",
     inertial_coords = grid_coords + amp * sin(omega * t);
   }
 }
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.FunctionOfTimeUpdater.Translation.2D",
```

Comment:
Could you factor the two tests into two functions (in an anonymous namespace) and have them be called from a single test case? This will reduce our total test run time :)